sickbeard/tv.py

```diff
@@ -2177,7 +2200,7 @@ def release_group(show, name):
             '%AB': '%(#)03d' % {'#': self.absolute_number},
             '%XAB': '%(#)03d' % {'#': self.scene_absolute_number},
             '%RN': release_name(self.release_name),
-            '%RG': rel_grp[relgrp],
+            '%RG': rel_grp[relgrp].upper(),
```

Comment:
%RG would just output the group name as it is in the file name (doesn't matter if capitalized or not), but the capitlized %RG should output the capitalized group name (like the other parameters). Now the user has the option to output it capitalized %RG or %rg lowercase.